submissions/zalevskyi/html-css-popup/input.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Pop-up CSS only</title>
+        <link rel="stylesheet" href="./css/popup-general.css">
+        <link rel="stylesheet" href="./css/popup-input.css">
+    </head>
+    <body>
+        <nav>
+            <div id="menu_main">
+                <a href="#" class="nav_text_link">About Us</a>
+                <input type="checkbox" id="menu_recycle_checkbox"/>
```

Comment:
When the user opened your popup focus state must be switch to the popup, in your case user continue to iterate through the rest menu items. [Watch video](https://share.vidyard.com/watch/BhR3Cu8hDmN9HhbSDe2ve8?)